package.json

```diff
@@ -28,9 +28,12 @@
   },
   "homepage": "https://github.com/ipfs/interface-ipfs-core#readme",
   "dependencies": {
+    "bl": "^1.1.2",
     "bs58": "^3.0.0",
     "chai": "^3.5.0",
-    "ipfs-merkle-dag": "^0.6.0"
+    "detect-node": "^2.0.3",
+    "ipfs-merkle-dag": "^0.6.0",
+    "readable-stream": "^2.1.4"
```

Comment:
We have to use ` "readable-stream": "^1.1.13",` cause version two introduced a issue when browserifying the code

https://github.com/ipfs/js-ipfs-unixfs-engine/blob/master/package.json#L59